### DISCUSSION TITLE
320x170 support?

### DISCUSSION THREAD

**Discussion by fwblack** [russhughes/st7789_mpy#94] [Ideas]
Hi;
Got it working on RPi Pico but the display I'm using is the Adafruit 1.9" 320x170 Color IPS TFT Display - ST7789
Product ID: 5394.
I first thought I could define my own rotations but I don't think that's the case.
Would it be possible to add native support for the 320x170 display?
Thanks,
Fred

**Comment by russhughes** [comment]
Yes, I'll add that.

**Reply by fwblack** [reply]
Thank you! Just tested and works.
Fred
